business/workloads.go

```diff
@@ -2104,5 +2216,35 @@ func (in *WorkloadService) streamParsedLogs(cluster, namespace, name string, opt
 
 // StreamPodLogs streams pod logs to an HTTP Response given the provided options
 func (in *WorkloadService) StreamPodLogs(cluster, namespace, name string, opts *LogOptions, w http.ResponseWriter) error {
-	return in.streamParsedLogs(cluster, namespace, name, opts, w)
+	return in.streamParsedLogs(cluster, namespace, name, opts, w, false)
+}
+
+// StreamZtunnelLogs streams pod logs to an HTTP Response given the provided options
+func (in *WorkloadService) StreamZtunnelLogs(cluster, namespace, name string, opts *LogOptions, w http.ResponseWriter) error {
+
+	// First, get ztunnel namespace and containers
+	pods := in.cache.GetZtunnel(cluster)
+	opts.PodLogOptions.Container = models.IstioProxy
+
+	// The ztunnel line should include the pod and the namespace
+	fs := filterOpts{
+		destWk: fmt.Sprintf("dst.workload=\"%s\"", name),
+		destNs: fmt.Sprintf("dst.namespace=\"%s\"", namespace),
+		srcWk:  fmt.Sprintf("src.workload=\"%s\"", name),
+		srcNs:  fmt.Sprintf("src.namespace=\"%s\"", namespace),
```

Comment:
This might need to include `dst.cluster` or `src.cluster` if ambient beings to support multi-cluster but I think this is fine for now for single cluster.

Reply:
We can create a follow up issue so we don't forget? Or maybe just a TODO